components/Storyblok/SbStory/SbStoryPage.tsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { SbStoryCardView, type SbStoryCardViewProps } from './SbStoryCardView';
-import { SbStoryFullView, type SbStoryFullViewProps } from './SbStoryFullView';
+import { SbStory, type SbStoryProps } from './SbStory';
 
-export type SbStoryPageProps = SbStoryCardViewProps & SbStoryFullViewProps & {
+export type SbStoryPageProps = SbStoryCardViewProps & SbStoryProps & {
   layout?: string;
 };
 
 export const SbStoryPage = (props: SbStoryPageProps) => {
   if (props.layout === 'story-card') {
     return <SbStoryCardView {...props} />;
   }
-  return <SbStoryFullView {...props} />;
+  return <SbStory {...props} />;
```

Comment:
I'll redo this when I get to Story Cards - feel like it's clumsy how this was handled because I didn't know any better 😂